scripts/linux/cross-platform-rust-build.sh

```diff
@@ -253,12 +333,7 @@ docker run --rm \
         cat /etc/os-release &&
 
         $SETUP_COMMAND
-
-        cd /project/$BUILD_PATH && 
-        echo 'Installing rustup' &&
-        curl -sSLf https://sh.rustup.rs | sh -s -- -y &&
-        . ~/.cargo/env &&
```

Comment:
Do we not need this? I think this will install the latest version of rust so it was needed even if the build agents have rust installed.

Reply:
I moved it inside each image.  The script to install for arm is slighty different and needs curl -sSLf https://sh.rustup.rs to be installed BEFORE $SETUP_COMMAND runs.
However I cannot really put it before (line 254, because curl has not been installed yet. The only solution was to move it inside $SETUP_COMMAND.
You can still see it line: 230 or 138 for example.

Reply:
I am confused because nothing changed with our arm builds and wasn't it working?